CommonTools/PileupAlgos/plugins/PuppiProducer.cc

```diff
@@ -95,6 +109,7 @@ void PuppiProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
     //Fill the reco objects
     fRecoObjCollection.clear();
     fRecoObjCollection.reserve(pfCol->size());
+    int ic = 0;
```

Comment:
given that `ic` is used across more than a few lines, perhaps a more descriptive variable name (or a code comment) can be helpful